ideas/000-discoverable-trusted-server-nodes/README.md

```diff
@@ -0,0 +1,73 @@
+---
+id: 000-discoverable-trusted-server-nodes
+title: Discoverable And Trusted Server Nodes
+status: Draft
+created: 2018-06-07
+category: core
+lead-contributor: adambabik
+contributors:
+exit-criteria: yes
+success-metrics: yes
+clear-roles: no
+future-iterations: yes
+roles-needed:
+    - Smart Contract Developer
+    - Go Developer
+    - Clojure Developer
+    - UI/UX Designer
+okrs:
+    - "[PO]: Objective: Messaging is reliable"
+    - "[P2]: KR: Cluster can handle 500 concurrent users"
+---
+
+## Summary
+The idea here is to provide a way to make Mail Server nodes discoverable and trusted. This should work seamlessly for the end user.
+
+## Product Overview
+Currently, Mail Servers (and other planned server nodes like Push Notification servers) need to be hardcoded in the app code. It's possible to provide your own Mail Server in the app settings but that's not a scalable solution and it does not solve all issues.
+
+The outcome of this swarm are fully decentralized and discoverable Mail Servers. The Status app will have a mean to verify that a discovered Mail Server is trusted.
+
+In the next iterations, we may work on a way to introduce external providers to submit their own Mail Servers into the cluster and allow end users to accept these resources as trusted.
```

Comment:
See TCRs, forked convo to Slack

Reply:
Link to Token-Curated Registries: https://docs.google.com/document/d/1BWWC__-Kmso9b7yCI_R7ysoGFIT9D_sfjH3axQsmB6E